NOTICE

```diff
@@ -67,8 +98,8 @@ Android AppCompat Library 1.4.1 maven:androidx.appcompat:appcompat:1.4.1
 	Copyright (C) 2021 The Android Open Source Project
 	Copyright 2018 The Android Open Source Project
 	Copyright 2021 The Android Open Source Project
-	Copyright (C) 2015 Google Inc.
 	Copyright 2017 The Android Open Source Project
+	Copyright (C) 2015 Google Inc.
 Android Arch-Runtime 2.1.0 maven:androidx.arch.core:core-runtime:2.1.0
 	Copyright (C) 2017 The Android Open Source Project  ~  ~ Licensed under the Apache License, Version 2.0 (the "License");  ~ you may not use this file except in compliance with the License.  ~ You
```

Comment:
There's extra text beyond copyright here in line 104.

Reply:
Fixed 👍🏻